lib/rubocop/cop/betterment/boolean_query_parameter.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Betterment
+      class BooleanQueryParameter < Base
+        MSG = <<~MSG
+          A boolean fetched from query params will never be false when
+          explicitly specified on the request. Please use
+          ActiveModel::Type::Boolean.new.cast to cast the value to a boolean.
```

Comment:
__question:__ just to be clear, we are ok with allowing "truthy" and "falsey" values to coerce to true/false?

The logic in ActiveModel::Type::Boolean can be found in: https://api.rubyonrails.org/classes/ActiveModel/Type/Boolean.html

I personally prefer to reject anything that doesn't match my exact expectations with a 400 error, but I do not know if we have any conventions to coerce.